docs/tutorial/offscreen-rendering.md

```diff
@@ -23,16 +24,28 @@ losses with no benefits.
 
 #### GPU accelerated
 
-GPU accelerated rendering means that the GPU is used for composition. Because of
-that, the frame has to be copied from the GPU which requires more resources,
-thus this mode is slower than the Software output device. The benefit of this
-mode is that WebGL and 3D CSS animations are supported.
+GPU accelerated rendering means that the GPU is used for composition. The benefit
+of this mode is that WebGL and 3D CSS animations are supported. There're two
+different approaches depends on whether `webPreferences.offscreenUseSharedTexture`
+is set to true.
+
+1. Use GPU shared texture
+
+    The frame are directly copied in GPU textures, thus this mode is very fast because
+    there's no CPU-GPU memory copies overhead and you can directly import the shared
+    texture to your own rendering program. The texture is passed in `texture` param of
+    `paint` event.
+
+2. Use CPU shared memory bitmap
+
+    The frame has to be copied from the GPU to the CPU bitmap which requires more
+    resources, thus this mode is slower than the Software output device. But is supports
+    GPU related functionalities.
```

Comment:
For (1), I think it should be mentioned that this is an advanced approach which requires a Node native module to handle the texture. For (2), it should mention that the texture is accessible using the `NativeImage` JavaScript API at the cost of performance.